deploy/chart/templates/plugin.yaml

```diff
@@ -205,6 +205,8 @@ spec:
 {{- end }}
             - mountPath: /host/run/csi-tool
               name: run-csi
+            - mountPath: /run/cnfs
```

Comment:
Adding host prefix path(/host) to  align with other host paths.

Reply:
You can think of it as similar to /var/lib/kubelet, and it won’t cause conflicts.

If we change this, we’ll also need to update the constant `defaultAlinasMountProxySocket = "/run/cnfs/alinas-mounter.sock"` in the Go code.